src/lib/web/logview.js

```diff
@@ -0,0 +1,39 @@
+import { Logger } from "../logger.js";
+export class Logview {
+	constructor({ host = process.env.WEB_HOST, key = process.env.WEB_KEY }) {
+		this.host = host || process.env.WEB_HOST;
+		this.key = key || process.env.WEB_KEY;
+		this.logger = new Logger({ callsystem: "web logview" });
+	}
+
+	async get(id) {
```

Comment:
why?

Reply:
the endpoint will not return json, it only currently returns an html page formatted for viewing

Reply:
> why?
 it would probably be better to expose the entire API as a class so we can eventually use it elsewhere in the app (i.e. callbacks and long responses)

Reply:
> the endpoint will not return json, it only currently returns an html page formatted for viewing

intentional behaviour, i wanted the helper to return the HTML page URL (somewhat because of design decisions but mostly to fix the trailing / thing)